index.js

```diff
@@ -114,40 +115,16 @@ function middleware (req, res, next) {
     }
   }
 
-  function _getAgent (userAgent, elements, callback) {
+  function _getAgent (userAgent, callback) {
   
     // If userAgent is undefined return browser: false
-    if(typeof userAgent === 'undefined')
-      return callback({browser: false, version: ''})
-
-    var regexps = {
-      'Chrome': [ /Chrome\/(\S+)/ ],
-      'Firefox': [ /Firefox\/(\S+)/ ],
-      'MSIE': [ /MSIE (\S+);/ ],
-      'Opera': [
-        /Opera\/.*?Version\/(\S+)/,     /* Opera 10 */
-        /Opera\/(\S+)/                  /* Opera 9 and older */
-      ],
-        'Safari': [ /Version\/(\S+).*?Safari\// ]
-      }
-      , re
-      , m
-      , browser
-      , version
-    if (typeof elements === 'undefined') {
-      elements = 2
-    }
-    else if (elements === 0) {
-      elements = 1337
-    }
-    for (browser in regexps) {
-      while (re = regexps[browser].shift()) {
-        if (m = userAgent.match(re)) {
-          version = (m[1].match(new RegExp('[^.]+(?:\.[^.]+){0,' + --elements + '}')))[0];
-          callback(null, {browser:browser, version:version})
-        }
-      }
+    if(typeof userAgent === 'undefined') {
+      return callback({browser: false, version: ''});
     }
+
+    var agent = ua.parse(userAgent);
```

Comment:
I think so, because we weren't able to catch all UserAgent from google & bingbot.
Even if this value is not used today, it will be used in the future.
We had the same feeling about the potential danger of not using this new value but benjamin explained to us its goal.